packages/models/scripts/seed.ts

```diff
@@ -226,11 +229,31 @@ async function main(): Promise<void> {
   const tutorManager = first(addedTutorManagers);
   if (!tutorManager) throw new Error("TutorManager not found; should never happen.");
 
-  await ratings.create({
-    raterId: student.id,
-    rateeId: tutor.id,
-    value: faker.number.int({ min: 1, max: 5 }),
-    feedback: faker.lorem.paragraph(1),
+  // seeding ratings data
+  await knex.transaction(async () => {
+    return await Promise.all(
+      students.map(async (student) => {
+
+        const randomValue = sample(range(1,6));
+        if (randomValue === undefined)
+          throw Error("Unexpected error: getting random rating value failed!");
+
+        const wordCount = sample(range(6, 20)) || 0;
+        const randomFeedback = faker.lorem.words(wordCount);
+
+        stdout.info(`Student ${student.id} is rating all available tutors.`);
+
+        const allTutors = [...addedTutors, ...addedTutorManagers];
+        for (const tutor of allTutors) {
+          await ratings.create({
+            raterId: student.id,
+            rateeId: tutor.id,
+            value: randomValue,
+            feedback: wordCount < 8 ? null : randomFeedback,
```

Comment:
With the current logic, the student will rate all tutors with the same number of starts and the same count. To increase randomnesses, let's create random rating & feedback for each tutor.